Makefile

```diff
@@ -0,0 +1,52 @@
+options:
+	@echo "You can use make with the following options: \n \
+    boot - Boot the deployments using up.sh.\n \
```

Comment:
How come you used `boot` as the command to make the deployment? Wouldn't it make sense to stick with `up`, since that's the common approach with Docker Compose.

Reply:
the reason is since this isn't really up but bootstrap. it would run up and then the bootstrap scripts but happy to change that if you want.